### PR TITLE
Update travis config to have sudo: false

### PR DESCRIPTION
**Pull request by amir20**
This fixes this error.

screen shot 2015-08-19 at 11 25 34 am

### CODE REVIEW

**Comment by amir20** [comment]
I have no idea why this build fails. It looks to be related to checkstyle. @karkum do you have any idea?

**Comment by amir20** [comment]
Yey it passes now. :) @karkum let me know if these changes don't make sense.

File: checkstyle.xml [hunk 1]
```diff
@@ -161,7 +161,6 @@
 
 
         <module name="EqualsAvoidNull"/>
-        <module name="RedundantThrows"/>
```
Comment (amir20) [inline checkstyle.xml]: Not sure why I had to do this. But it is not supported anymore. I think we should update our checkstyle rules to new opower checkstyle. I thought upgrading `opower-parent` should do that, but I guess not.

**Comment by karkum** [comment]
Looks good! Merge away

**Pull request merged**